dag_in_context/src/schema.egg

```diff
@@ -240,3 +240,8 @@
 (bop->string (PtrAdd) "PtrAdd")
 (bop->string (Print) "Print")
 (bop->string (Free) "Free")
+
+
+;; If anything is put in the DebugExpr relation, we'll extract them instead of the original program.
```

Comment:
Is there a reason you want to do it instead of the original program not in addition?

Reply:
Actually you do want it in addition so you can resolve calls- PR for that coming soon